enrolments/models.py

```diff
@@ -67,3 +67,7 @@ class Enrolment(models.Model):
         blank=True,
         on_delete=models.SET_NULL,
     )
+
+    def clean(self):
+        validate_enrolment(self)
+        return super().clean()
```

Comment:
nit: can we get a new line at the end of file?

Reply:
hmm for some reason it looks fine in my ide but not on github... I think it should still be there?